File: WebContent/js/service/userService.js
//服务层
app.service('userService',function($http){
	this.findUser=function(){
		return $http.get('user/user.do');
	}
	
	this.addUser=function(user){
		return $http.post('user/register.do',user);
	}
	
	
	this.findMe=function(user){
		return $http.post('user/findeMe.do',user);
	}
	
	this.updatePassword=function(user){
		return $http.post('user/updatePassword.do',user);
	}
	
	this.updateUserInfo=function(entity){
		return $http.post('user/updateUserInfo.do',entity);
	}
});

File: src/com/family/service/UserService.java
package com.family.service;

import com.family.domain.Result;
import com.family.domain.User;

public interface UserService {

	User findOne(String username);

	User findById(String string);

	void addUser(User user);

	void updateUser(User user);

	void updatePassword(User user);

	Result findMe(User user);

}
